server/app.js

```diff
@@ -223,8 +226,8 @@ module.exports = function createApp({ signInService }, logger, calendarService,
 
   // Routing
   app.use('/', standardRoute(createLoginRouter()))
-  app.use('/calendar', authHandler, standardRoute(createCalendarRouter(logger, calendarService, notificationService)))
-  app.use('/details', authHandler, standardRoute(createCalendarDetailRouter(logger, calendarService)))
+  app.use('/calendar', authHandler, standardRoute(createCalendarRouter(logger, calendarService, notificationService, userAuthenticationService)))
+  app.use('/details', authHandler, standardRoute(createCalendarDetailRouter(logger, calendarService, userAuthenticationService)))
   app.use('/notifications', authHandler, standardRoute(createNotificationRouter(logger, notificationService)))
```

Comment:
why does `/notifications` not need the user authentication code too?

Reply:
ah because you don't need the notification api url from it

Reply:
Yes it does, this is missed by me. I'll fix.